src/test/java/uk/gov/pay/connector/events/resource/EmittedEventResourceTest.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.pay.connector.events.resource;
+
+import io.dropwizard.testing.junit.ResourceTestRule;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.pay.connector.events.EmittedEventsBackfillService;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Response;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+
+@RunWith(MockitoJUnitRunner.class)
+public class EmittedEventResourceTest {
+    private static final EmittedEventsBackfillService emittedEventsBackfillService = mock(EmittedEventsBackfillService.class);
+    
+    @ClassRule
+    public static final ResourceTestRule resources = ResourceTestRule.builder()
+            .addResource(new EmittedEventResource(emittedEventsBackfillService))
+            .build();
+
+    @Test
+    public void shouldReturn200() {
+        Response response = resources
+                .target("/v1/tasks/emitted-events-sweep")
+                .request()
+                .post(Entity.json(""));
+
+        assertThat(response.getStatus(), is(200));
```

Comment:
[minor] for readability
```suggestion
        assertThat(response.getStatus(), is(HttpStatus.SC_OK/Status.OK));
```

Reply:
Will update in follow up PR.